### PR TITLE
Updated PortX.app version 1.1.4

### PR DESCRIPTION
**Pull request by netsarang-bigseungho**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

After making all changes to a cask, verify:

- [ ] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [ ] `brew audit --cask <cask>` is error-free.
- [ ] `brew style --fix <cask>` reports no offenses.

Additionally, **if adding a new cask**:

- [ ] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- [ ] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [ ] Checked the cask is submitted to [the correct repo](https://docs.brew.sh/Acceptable-Casks#finding-a-home-for-your-cask).
- [ ] `brew audit --new-cask <cask>` worked successfully.
- [ ] `brew install --cask <cask>` worked successfully.
- [ ] `brew uninstall --cask <cask>` worked successfully.

### CODE REVIEW

File: Casks/portx.rb [hunk 1]
```diff
@@ -1,16 +1,16 @@
 cask "portx" do
-  version "1.1.4,10.2"
-  sha256 :no_check
+  version "1.1.4"
+  sha256 "2c612c434fa76b943af6d583293354063b3180334d976bcb804ade1606bbbd07"
 
-  url "https://cdn.netsarang.net/0ac7ea20/PortX-latest.dmg",
+  url "https://cdn.netsarang.net/0ac7ea20/PortX-#{version}.dmg",
```
[the remark this answers is not in the document]
Comment (NetSarangComputer) [reply Casks/portx.rb:5]: The download URL specified by the version. It's not damaged, so please check it again.

**Comment by NetSarangComputer** [comment]
Sir, why can't I keep crossing this section? brew audit --cask --appcast --online './Casks/portx.rb'

**Comment by NetSarangComputer** [comment]
I'd appreciate it if you could give me a solution.

**Comment by bevanjkay** [comment]
@NetSarangComputer This is because the livecheck is failing. The best option is probably to revert the cask to before this PR was submitted. The cask was already at version `1.1.4` and have a working livecheck.

**Comment by NetSarangComputer** [comment]
Oh... Does that mean I need to revert to the previous version and update 1.1.4?

**Comment by bevanjkay** [comment]
@NetSarangComputer I think you can just close this PR for now, as the cask was already pointing to the latest version.

**Comment by NetSarangComputer** [comment]
Thank you for your help, sir.

**Pull request closed without merging**